### ISSUE TITLE
If you have a line with tabs it causes a bug in infix_spaces_linter

### ISSUE THREAD

**Issue by jimhester** [jimhester/lintr#44]
``` r
    if(a == b) {
    1
}
```

```
bad.R:1:1: style: Use two spaces to indent, never tabs.
    if(a == b) {
^
bad.R:1:11: style: Place a space before left parenthesis, except in a function call.
    if(a == b) {
          ^
bad.R:1:14: style: Put spaces around all infix operators.
    if(a == b) {
            ~^~~
```

The arrows are lined up properly on my machine, but there should not be any lint for infix operators in this case.

**Comment by kgryte** [comment]
What is the timeline for pushing these (and other changes) to CRAN? Working with a codebase which uses tab indentation and seeing the false positives noted above.
